content-src/components/DiscoveryStreamComponents/DSCard/DSCard.jsx

```diff
@@ -8,9 +8,9 @@ export class DSCard extends React.PureComponent {
           <div className="img" style={{backgroundImage: `url(${this.props.image_src}`}} />
         </div>
         <div className="meta">
-          <header>{this.props.title}</header>
-          <p>{this.props.excerpt}</p>
-          <p>{this.props.source}</p>
+          <header className="title">{this.props.title}</header>
```

Comment:
This is probably fine, but there is another component that uses the classname title (topsites) but I think in both cases and css styling has proper specificity in the css as to not leak. In your case it's `.ds-card .meta .title`